loadtest/tests.go

```diff
@@ -603,6 +603,38 @@ func actionGetTeamUnreads(c *EntityConfig) {
 	}
 }
 
+func actionGetChannelUnreads(c *EntityConfig) {
+	user, resp := c.Client.GetMe("")
+	if resp.Error != nil {
+		mlog.Error("Failed to get me", mlog.Err(resp.Error))
+		return
+	}
+
+	team, channel := c.UserData.PickTeamChannel(c.r)
+	if team == nil || channel == nil {
+		return
+	}
+
+	channelId, err := c.GetTeamChannelId(team.Name, channel.Name)
+	if err != nil {
+		mlog.Error("Unable to get channel from map", mlog.String("team", team.Name), mlog.String("channel", channel.Name), mlog.Err(err))
+		return
+	}
+
+	if rand.Float64() < c.LoadTestConfig.UserEntitiesConfiguration.GetPostsAroundLastUnreadChance {
+		numPosts := c.LoadTestConfig.UserEntitiesConfiguration.NumGetPostsAroundLastUnread
+		_, resp := c.Client.GetPostsAroundLastUnread(channelId, user.Id, numPosts, numPosts)
+		mlog.Info("Failed to get posts around last unread", mlog.String("channel_id", channelId), mlog.Err(resp.Error))
```

Comment:
shouldn't there be an `if` here? you don't seem to be checking `resp` for error

Reply:
Indeed, great catch :+1: